packages/radio-player/demo/radio-player-controller.ts

```diff
@@ -48,15 +40,54 @@ export default class RadioPlayerController extends LitElement {
     `;
   }
 
-  async loadItemMetadata() {
+  /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
In general, I like this eslint rule because it enforces typing wherever possible. In this case, I'm dealing with the generic metadata JSON blob instead of a proper model so I have to use `any`.

This is a case where I'd like to flesh out our data models more. I'd like to come up with a full on data model for our metadata and pass that around instead of the JSON blob. This would let me get rid of this eslint disabling.